### PR TITLE
Add run command

### PR DESCRIPTION
**Pull request by satyarohith**
Execute slash commands in Mattermost.

execrun

Can be merged once commander supports `...` as a parameter name.

### CODE REVIEW

**Comment by niks3089** [comment]
Can you create a PR for slack separately? I would like to merge this soon

**Comment by satyarohith** [comment]
Sure. Need to push some error handling and stuff. And once the `commands.yaml` file issue is fixed, we can merge it.

File: run/run.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+ * @param {?string} commandText text message
+ * @param {!object} [secrets = {}] list of secrets
+ * @return {Promise<SlackBodyType>} Response body
+ */
+async function _command(params, commandText, secrets = {}) {
+  const {mmURL, mmToken} = secrets;
+  if (!mmURL || !mmToken) {
+    return {
+      response_type: 'ephemeral',
+      text: `You need to create secrets named \`mmURL\` & \`mmToken\` with your mattermost server URL & API token. Create secrets by running \`/nc secret_create\``
```
Comment (niks3089) [inline run/run.js]: Split this into multiple lines.

Comment (satyarohith) [reply run/run.js]: Will mark ready for review soon. Need to fix quite a few things.

**Comment by niks3089** [comment]
What’s the status of this PR? Can we add this to readme as well?

**Comment by satyarohith** [comment]
Yes, but it only supports Mattermost.

**Comment by satyarohith** [comment]
@niks3089  I've updated the docs.

**Pull request merged**